asdf/tests/test_api.py

```diff
@@ -352,38 +354,36 @@ def test_extension_version_check(installed, extension, warns):
 
 def test_auto_inline(tmpdir):
     outfile = str(tmpdir.join('test.asdf'))
-    tree = {"small_array": np.arange(6), "large_array": np.arange(100)}
+    small = 75
```

Comment:
Maybe this should be defined relative to DEFAULT_AUTO_INLINE as well?

Reply:
I set it to DEFAULT_AUTO_INLINE-1